### PR TITLE
Performance Profiler: Improve LLM recommendations area

### PR DESCRIPTION
**Pull request by WBerredo**
Related to [Performance Profiler: LLM implementation](https://github.com/Automattic/dotcom-forge/issues/8825)

## Proposed Changes

* Update the feature flag for LLM to true as this is the default behavior now
  * I've turned this feature to true even in environments where the Performance Profiler is disabled as this should already be the default behavior when it gets activated
*  Show a messaging with a rotating icon when we are loading an IA message 
* Show a message stating the content was generated with IA 
* Show a survey in the end
  * Currently the survey is generating a track event

## Why are these changes being made?

To better display LLM messages generated

## Testing Instructions

* Go to `/speed-test-tool?url=:site`. Ex: `/speed-test-tool?url=wordpress.com`
* Go to the `Recommendations` section
* Click on a item and see if the `Loading state` is properly displayed
* After loading check if the answer has the two sections: `"Why is this important?"` and `"How to fix this?"`
* Check if there is a box below saying `Generated with IA`

CleanShot 2024-09-05 at 16 53 57

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

**Review by gavande1 — APPROVED** [review]
Works great. Thank you for the hard work @WBerredo ! 🙇

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/16653162

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @WBerredo for including a screenshot in the description! This is really helpful for our translators.

File: client/performance-profiler/components/llm-message/style.scss [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	gap: 10px;
+
+	.content {
+		display: flex;
+		align-items: center;
+		gap: 8px;
+	}
+
+	.rotate {
+		animation: rotation 3s infinite linear;
```
Comment (matt-west) [inline client/performance-profiler/components/llm-message/style.scss:26]: @WBerredo Big Sky has an animated loading icon we can use here. I’ll drop you the file via Slack as GH might over-optimise it.

ai-loading

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/16653162).

**Pull request merged**